src/lib/viewers/media/Dash.scss

```diff
@@ -1,12 +1,17 @@
 @import 'mediaBase';
 
+.bp-media-container {
+    display: flex;
+    justify-content: center;
+    min-width: 320px; // Prevent media controls from overflowing on small screens
```

Comment:
Should this be in  `_mediaBase.scss` instead?

Reply:
I'll check to see if the issue affects MP4 files. I didn't actually realize they had a separate viewer until now.